### DISCUSSION TITLE
how to save files

### DISCUSSION THREAD

**Discussion by krishno16** [Cyfrin/foundry-full-course-cu#2422] [General]
hey can you please tell me how can i save my work which i am doing in remix 
how can i save those files

**Comment by EngrPips** [comment]
I don't really bother about saving files in remix as it is meant to be used for quick development to understand the basics of smart contract development. You should move to a local environment for development if you want to start building anything serious.

**Comment by dharminnagar** [comment]
Don't think about saving the work you do in remix. If you're in the `Solidity 101` course and thinking about how to revisit those things, don't worry when you'll start the `Foundry 101` course Patrick takes the same project to build in local development.
Remix is just a quick IDE to check some things and quickly spin up a project, it's not meant for full-scale development

**Comment by EngrPips** [comment]
Indeed
